### PR TITLE
doc: Add Bluetooth host changes to 1.9 release note

### PR DESCRIPTION
**Pull request by Vudentz**
Signed-off-by: Luiz Augusto von Dentz <luiz.von.dentz@intel.com>

### CODE REVIEW

File: doc/release-notes-1.9.rst [hunk 1]
```diff
@@ -74,7 +74,9 @@ Networking
 Bluetooth
 *********
 
-* change description
+* Bluetooth Mesh support (all mandatory features and most optional ones)
+* GATT Service Changed Characteristic support
+* IPSP net-app support
```
Comment (dbkinder) [inline doc/release-notes-1.9.rst]: Might expand on this more, something like:

     * IPSP net-app support: a simplified networking API reducing duplication of common
       tasks an application writer has to go through to connect to the network

**Review by dbkinder — CHANGES_REQUESTED** [review]
a bit more detail would be better for the release notes

**Comment by Vudentz** [comment]
I incorporate the changes suggested.

**Review by jhedberg — APPROVED** [review]

**Comment by carlescufi** [comment]
@dbkinder I've added the controller-side release notes. I haven't added more detail in purpose, to align them with the other release notes from other subsystems

**Review by nashif — APPROVED** [review]

**Pull request merged**